ubuntudesign/www.ubuntu.com
# Phone scss tidy

## Done

* a general clean-up of the _phone.scss file
* minor design changes where bugs were spotted or when @yaili spotted something
 * bug on helping hand section on /phone/developer ![image](https://cloud.githubusercontent.com/assets/441217/18888482/5d274b98-84ad-11e6-9e33-b4f23bded055.png)
  * bug on form on /phone/devices medium 
![image](https://cloud.githubusercontent.com/assets/441217/18888529/9398457e-84ad-11e6-848c-f453306bac03.png) small 
![image](https://cloud.githubusercontent.com/assets/441217/18888537/9ed01eee-84ad-11e6-8428-8692c77eb7ab.png)
* removed unused _phone_promo.html file
* used columns for better medium and small screen display of form in _devices_newsletter.html 

## QA

1. go to every page in the cloud section and see that it looks nearly identical to live
